### ISSUE TITLE
Unpublished notes / reason / the "why"

### ISSUE THREAD

**Issue by dgershman** [bmlt-enabled/bmlt-server#82]
When a meeting is unpublished, it's for a variety of reasons.  Being able to document the "why" in a notes field of some sort and then have that exported to NAWS will help greatly.

**Comment by radius314** [comment]
This would ideally have some canned responses.

**Comment by pjaudiomv** [comment]
Yes, maybe even force to make a selection or type something. I administer two servers and often get people who should be deleting meetings rather than unpublishing. However I can't always be sure so this would be super helpful in general not just to naws. Obviously some of that can be helped by better training on using the server.

**Comment by ChrisMarshallNY** [comment]
I just had a bit of back-and-forth with Danny on this. I think I understand what we need, and have a couple of ideas.

The main deal is to try to avoid a DB schema change.

With that in mind, we could repurpose the `published` column. It is currently a `tinyint(4)`, which carries either `1` (published) or `0` (unpublished -default).

We could create an enum, based on int values, which map to concrete classes of reasons, like so:

`0 - unpublished (no reason given)`
`1 - published`
`2 - unpublished - moving venue`
`3 - unpublished - seasonal`
`4 - unpublished - Crazy Joe threw a tantrum in the parking lot, and the meeting is taking some "down time."`

etc...

**Comment by ChrisMarshallNY** [comment]
We might also be able to use the `change_name_string`. It is a rather tenuous field (and it also has a `description`). I suspect I don't use them much. These are the change record; not the meeting data.

**Comment by pjaudiomv** [comment]
hidden could be better term than unpublish ? prob could use better UI indicator for unpublished (right now meetings just turn yellow)
